docusaurus/docs/Android/02-client/06-guides/01-push-notifications/01-overview.mdx

```diff
@@ -35,14 +35,30 @@ ChatClient.Builder("api-key", context)
     .build()
 ```
 
+:::note
+You can disable push notifications client-side by providing custom `NotificationConfig` with `pushNotificationsEnabled` set to `false`.
+Keep in mind that the SDK creates a `NotificationChannel` during its initialization if push notifications are enabled client-side.
+:::
+
 ## Customizing Push Notifications
 
 If you want, you can also customize how the push notifications work.
 
-There are two ways to customize push notifications:
+There are three ways to customize push notifications:
+* Overriding resources: Lets you customize notification's icon and text resources
 * `NotificationHandlerFactory`: Uses the styles we provide but customizes an intent the user triggers when clicking on a notification.
 * `NotificationHandler`: Lets you fully customize how notifications are shown and dismissed
 
+### Overriding resources
+
+You can provide custom icon and text resources displayed in notifications by overriding them:
+- We are using [stream_ic_notification.xml](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-client/src/main/res/drawable/stream_ic_notification.xml) drawable as the default notification's icon
+- Text resources are placed inside [string.xml](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-client/src/main/res/values/strings.xml) file
```

Comment:
It'd feel nicer to me if these were some kind of runtime APIs where you can provide these values instead of resource overrides. This feels a bit too rigid.

Reply:
It is following the same approach we are using for "custom translations" of our SDK

Reply:
I know it is, I just don't love it, especially when we extend it beyond strings (to drawables, in this case). Actual methods in code are a lot more obviously public API than resources. Making changes to resources without realizing that it's a breaking change for clients is unfortunately much, much easier.

That being said, we can move ahead with this PR, just wanted to voice that opinion.